logging/bin/deploy_logging_opensearch.sh

```diff
@@ -0,0 +1,115 @@
+#! /bin/bash
+
+# Copyright © 2020, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# identify backend search provider
+export LOG_SEARCH_BACKEND="OPENSEARCH"
+
+cd "$(dirname $BASH_SOURCE)/../.."
+source logging/bin/common.sh
+
+# Confirm NOT on OpenShift
+if [ "$OPENSHIFT_CLUSTER" == "true" ]; then
+  if [ "${CHECK_OPENSHIFT_CLUSTER:-true}" == "true" ]; then
+    log_error "This script should NOT be run on OpenShift clusters"
+    log_error "Run logging/bin/deploy_logging_open_openshift.sh instead"
+    exit 1
+  fi
+fi
+
+
+# set flag indicating wrapper/driver script being run
+export LOGGING_DRIVER=true
+
+##################################
+# Check pre-reqs                 #
+##################################
+
+checkDefaultStorageClass
+
+# Create namespace if it doesn't exist
+if [ "$(kubectl get ns $LOG_NS -o name 2>/dev/null)" == "" ]; then
+  kubectl create ns $LOG_NS
+fi
+
+
+log_notice "Deploying logging components to the [$LOG_NS] namespace [$(date)]"
+
+set -e
+
+##################################
+# Event Router                   #
+##################################
+
+# Collect Kubernetes events as pseudo-log messages?
```

Comment:
No, that is explaining what the Event Router will do if it is enabled.  But will reword or remove and push since we don't do that for the other components being deployed.